### DISCUSSION TITLE
ComboBox popup scrolling

### DISCUSSION THREAD

**Discussion by nununoisy** [slint-ui/slint#2917] [Q&A]
When I have a ComboBox widget with a lot of items, the popup tends to get cut off by the window or screen edge, and items below that cutoff are not easily accessible (you can only select them by mashing the down arrow key).

Is there any way to limit the size of the popup so users can scroll through the items?

[SlintPad MRE](https://slint.dev/releases/1.0.2/editor/?snippet=import+%7B+ComboBox+%7D+from+%22std-widgets.slint%22%3B%0Aexport+component+Example+inherits+Window+%7B%0A++++width%3A+200px%3B%0A++++height%3A+130px%3B%0A++++ComboBox+%7B%0A++++++++y%3A+0px%3B%0A++++++++width%3A+self.preferred-width%3B%0A++++++++height%3A+self.preferred-height%3B%0A++++++++model%3A+%5B%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%2C%0A++++++++++++%220%22%2C+%221%22%2C+%222%22%2C+%223%22%2C+%224%22%2C+%225%22%2C+%226%22%2C+%227%22%2C+%228%22%2C+%229%22%0A++++++++%5D%3B%0A++++%7D%0A%7D%0A)

**Comment by tronical** [comment]
I think this is the same bug as #2375 (unfortunately:-(

**Reply by nununoisy** [reply]
Seems like it. Atp the options seem to be either wait for #2375 or write a custom widget using a ScrollView internally. Thanks for the reply!
